CONTRIBUTING.md

```diff
@@ -113,6 +113,11 @@ following rules before you submit a pull request:
    scale in dimensionality: n_features is expected to be lower than
    100".
 
+-  If your PR includes changes to files under the ``doc/`` or
```

Comment:
No, we use this logic automatically.

What we need to be saying is more along the lines of:
* describing the roles of the different CIs we use: travis runs on various linux settings, appveyor on various windows settings, both do some coverage analysis, and CircleCI builds the documentation for viewing (for which a URL is given in `contributing.rst`)
* noting that `[ci skip]` appearing in the latest commit message will stop these running
* noting that while some heuristics are used for Circle to decide between running a full HTML documentation build, including plots (`make html`) and a quick one (`make html-noplot`), putting one of `[doc build]`, `[doc quick]` and `[doc skip]` will override this logic with the build including example gallery plots, the build excluding example gallery plots, and no documentation build.